### PR TITLE
Add example-workflows

### PR DESCRIPTION
**Pull request by highway7**
Added an example workflow of Github Actions for deploying a python script to Google Cloud Functions.

### CODE REVIEW

File: example-workflows/.github/workflows/cloud-functions.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      with:
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+        export_default_credentials: true
+    - id: Deploy
+      uses: google-github-actions/deploy-cloud-functions@main
+      with:
+        name: helloworld-python
+        runtime: python37
+        entry_point: hello_world
+
```
Comment (bharathkkb) [inline example-workflows/.github/workflows/cloud-functions.yml:47]: nit: new line

File: example-workflows/.github/workflows/cloud-functions.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    # Setup gcloud CLI
+    - uses: google-github-actions/setup-gcloud@master
+      with:
+        service_account_key: ${{ secrets.GCP_SA_KEY }}
+        export_default_credentials: true
+    - id: Deploy
+      uses: google-github-actions/deploy-cloud-functions@main
```
Comment (bharathkkb) [inline example-workflows/.github/workflows/cloud-functions.yml:42]: As this is a native action, we should prefer authenticating natively via `credentials` if we are not using gcloud in any subsequent steps.

File: example-workflows/main.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        Response object using
+        `make_response <http:#flask.pocoo.org/docs/1.0/api/#flask.Flask.make_response>`.
+    """
+    request_json = request.get_json()
+    if request.args and 'message' in request.args:
+        return request.args.get('message')
+    elif request_json and 'message' in request_json:
+        return request_json['message']
+    else:
+        return f'Hello World!'
```
Comment (bharathkkb) [inline example-workflows/main.py:30]: nit: newline

File: example-workflows/README.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
+# Cloud Functions - GitHub Actions
+
+An example workflow that uses [GitHub Actions][actions] to deploy a
```
Comment (bharathkkb) [inline example-workflows/README.md:3]: Can we link to the workflow.yml here

**Review by bharathkkb — CHANGES_REQUESTED** [review]
Thanks for the PR! This looks great, some minor nits

**Comment by highway7** [comment]
Will update and submit PR again.

**Pull request closed without merging**